docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
```

Comment:
I think it'd be really useful to identify the "upgrade flow" that we suggest that users follow and the semantics of which versions are compatible in which others for how long.

For example, we suggest that users upgrade their router version before upgrading their build pipeline version before updating their subgraph schemas / libraries. We guarantee that minor versions of router will continue to work with supergraph schemas that were built with build versions that are 2 minor below, etc.

Is that documented anywhere yet?

Reply:
I agree and believe the answer is no, and in fact the "Moving to Fed 2" article (https://www.apollographql.com/docs/federation/federation-2/moving-to-federation-2) doesn't really touch on managed federation considerations at all. This will take some extra lift so I'll create a separate issue for it.